scripts/x-charts.exports.json

```diff
@@ -73,6 +75,12 @@
   { "name": "LineElementOwnerState", "kind": "Interface" },
   { "name": "LineElementPath", "kind": "Variable" },
   { "name": "LineElementProps", "kind": "TypeAlias" },
+  { "name": "LineHighlightElement", "kind": "Function" },
+  { "name": "lineHighlightElementClasses", "kind": "Variable" },
+  { "name": "LineHighlightElementClasses", "kind": "Interface" },
+  { "name": "LineHighlightElementOwnerState", "kind": "Interface" },
```

Comment:
Is there a point in exporting the `OwnerState` type in charts codebase? 🤔 
I don't see it exported on pickers or data grid. 🤷

Reply:
Not really, I probably added export by reflex when defining those interfaces